cfg/sequencer.cfg

```diff
@@ -49,7 +49,7 @@ check_dl1: lstchain_check_dl1
 dl1_to_dl2: lstchain_dl1_to_dl2
 
 # Path to longterm dl1 script
-longterm_check: /fefs/aswg/software/virtual_env/ctasoft/cta-lstchain/lstchain/scripts/longterm_dl1_check.py
+longterm_check: /fefs/aswg/software/virtual_env/ctasoft/cta-lstchain/lstchain/scripts/lstchain_longterm_dl1_check.py
```

Comment:
We can remove this path from here and directly use the lstchain entry point `lstchain_longterm_dl1_check` in `closer.py`